packages/tgpu-cli/tgpu-cli.mjs

```diff
@@ -21,32 +27,52 @@ const args = arg({
   '-w': '--watch',
 });
 
+const ALLOWED_EXTENSIONS = ['.js', '.cjs', '.mjs', '.ts', '.cts', '.mts'];
+
 const COMMANDS = {
   gen: {
     help: () =>
       console.log(`\
-Generate a ts file from a wgsl file.
+Generate a ts/js file from a wgsl file.
 
 Usage:
-  tgpu-cli gen --input <input> [--output <output>] [--watch]
-  tgpu-cli gen <input> [--output <output>] [--watch]
+  tgpu-cli gen --input <input> --output <output> [--watch] [--commonjs]
+  tgpu-cli gen <input> --output <output> [--watch] [--commonjs]
 
 Options:
-  --input, -i   The input file or glob pattern.
-  --output, -o  The output file. If not provided, the input file will be used with a .ts extension. Cannot be used with glob patterns.
-  --watch, -w   Watch for changes in the input file(s) and regenerate the output file(s)`),
+  --input, -i\t The input file or glob pattern.
+  --output, -o\t The output name or pattern for generated file(s).
+  --watch, -w\t Watch for changes in the input file(s) and regenerate the output file(s).
+  --commonjs\t Generate a CommonJS style file.
+`),
+
     execute: async () => {
       const input = args['--input'] ?? args._[1];
       const output = args['--output'];
```

Comment:
Now that output is not optional maybe we could also make it a positional argument for _convenience_.

Reply:
wouldn't that be confusing which argument should come first?

Reply:
I don't think so but i'm not 100% on that. Maybe if someone is not sure he can use `--help`. I will take a look how other cli's handle that and get back to you.

Reply:
now that I think of it I also believe people may expect that typing multiple file names would result in both of them being inputs, not that the second one is the input